privacyidea/static/templates/header.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="Cornelius Kölbel" >
-    <link rel="icon" href="../../favicon.ico">
+    <link rel="icon" href="static/templates/favicon.ico">
```

Comment:
The path should be prefixed with the `{{ instance }}` variable in case privacyIDEA does not run in the web root.
Also it would make sense to put the icon in the `static`-folder since it is not a template.